### ISSUE TITLE
Robot's locked slots

### ISSUE THREAD

**Issue by nixus-777** [TerryCavanagh/diceydungeons.com#2513]
It would be better if functionality of an equipment with locked slots could be shown more often. For example, hovering at equipment in the inventory, viewing equipment in the anvil (which is useful for modding, since equipment that changes functionality and has locked slots can be understood). And also, equipment with locked slots always has a regular slot for a dice, which is not very useful. I have a suggestion of how locked equipment can be displayed (I apologize for the quality of the scheme, but the idea is important here). Also, whether the equipment is locked or not can be affected by an equipment's tag.
![Безымянный](https://user-images.githubusercontent.com/89795491/131368674-dd4c0afc-8012-48e6-af90-02d5afc58692.png)

**Comment by TerryCavanagh** [comment]
Yep; this would be a nice thing to improve! Locked equipment is a little under developed as a concept, I think.
